airflow/bin/cli.py

```diff
@@ -910,11 +927,36 @@ class CLIFactory:
             'args': ('roles',),
         },
     )
+    CONFIG_COMMANDS = (
+        {
+            'name': 'show',
+            'func': lazy_load_command('airflow.cli.commands.config_command.show_config'),
+            'help': 'Show current application configuration',
+            'args': ('section',),
+        },
+        {
+            'name': 'set',
+            'func': lazy_load_command('airflow.cli.commands.config_command.set_config_option'),
+            'help': 'Set an option in config: '
+                    'airflow config set --section=core.executor --value=LocalExecutor',
+            'args': ('section', 'value', 'option', 'executor', 'dagdir'),
+        },
+        {
+            'name': 'location',
```

Comment:
We can use `cfg_path` like in `run` subcommand

Reply:
You mean `airflow config cfg_path`?

Reply:
Sounds reptitive... Naah, I think location is better. Or `file_path`.

No opinion on that

Reply:
Maybe `airflow config path`